.changeset/good-papers-shake.md

```
@@ -0,0 +1,11 @@
+---
+'@mastra/mongodb': patch
+'@mastra/spanner': patch
+'@mastra/core': patch
+'@mastra/libsql': patch
+'@mastra/mssql': patch
+'@mastra/mysql': patch
+'@mastra/pg': patch
+---
+
+Fixed schedules of dynamically created workflows disappearing after a restart. Their schedule configuration is now persisted and restored, so the Schedules tab in Studio keeps showing them and they continue to run.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)